src/components/common/Header/Header.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import Header from './Header';
+
+const meta: Meta<typeof Header> = {
+  title: 'common/Header',
+  component: Header,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof Header>;
+
+export const Defualt: Story = {
+  args: {
+    children: 'Header',
+    variants: 'primary',
+    size: 'md',
+  },
+};
```

Comment:
args（というかprops）はHeaderにはないはずだから空で大丈夫！
```suggestion
export const Defualt: Story = {};
```